modules/azure/master-as/master-as.tf

```diff
@@ -93,6 +93,11 @@ resource "azurerm_virtual_machine" "master" {
     }
   }
 
+  boot_diagnostics {
```

Comment:
Do we really need boot diagnostics? we are experts already :trollface:

Reply:
Well, it gives us logs for instance if something fuck ups. If nothing breaks we don't need it. 
I am fine with removing it.

Reply:
removed fro now